pythagorean_thm.py

```diff
@@ -0,0 +1,27 @@
+'''
+This script will help solve the pythagorean theorem.
```

Comment:
What's the Pythagorean Theorem?

Reply:
Should this include a few paragraphs on the Life and Times of Pythagoras?

Reply:
haha Craig